lib/global.js

```diff
@@ -1,14 +1,3 @@
-var createTorrent = require('create-torrent')
-var wrtc = require('wrtc')
+var wrtc = require('electron-webrtc')()
 
 global.WRTC = wrtc
-
-global.WEBTORRENT_ANNOUNCE = createTorrent.announceList
```

Comment:
Okay, its a bit confusing that all of the webtorrent projects use the [create-torrent](https://github.com/feross/create-torrent) module and each of them still handles this restriction by themself. What do you recommend, reverting the removal here or adding a reference at the shared source ?